vizro-ai/docs/pages/user-guides/genAI-use-case.md

```diff
@@ -0,0 +1,105 @@
+# How to add Vizro-AI created charts into a Vizro dashboard
+
+This guide provides instructions on incorporating a chart generated by Vizro-AI into your [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
+
+1. Create a chart with Vizro-AI that you would like to visualize in a dashboard.
+
+    In this example, we aim to create a chart that illustrates the population development of each continent over time. To gain deeper insights and access the underlying code responsible for generating the chart, include `explain=True` in the `plot()` method. Let's execute the provided code and examine the outcome.
```

Comment:
I think you should include `explain=True` it in the code snippet below since you mention it here.

Reply:
Good catch. 🙌